crowbar_framework/app/controllers/barclamp_controller.rb

```diff
@@ -19,6 +19,7 @@
 require "json"
 
 class BarclampController < ApplicationController
+  skip_before_filter :enforce_installer
```

Comment:
https://github.com/crowbar/crowbar-core/commit/faf9bc1e79a357a942fef83da8d85d325751144c should hopefully fix that issue